src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,35 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int positionOfDate = 0;
+    private static final int positionOfName = 1;
+    private static final int positionOfHours = 2;
+    private static final int positionOfSalary = 3;
```

Comment:
These are constants, change the name's style.

Reply:
Thank you for responding so quickly. The name's style were changed.